src/registrar/assets/js/get-gov-admin.js

```diff
@@ -551,27 +566,25 @@ function initializeWidgetOnList(list, parentId) {
                 let emailBody = emailData.email_body_text
                 if (emailBody) {
                     actionNeededEmail.value = emailBody
-                    showActionNeededEmail(actionNeededEmail);
+                    showActionNeededEmail(actionNeededEmail, noEmailMessage);
                 }else {
-                    showNoEmailMessage(actionNeededEmail);
+                    showNoEmailMessage(actionNeededEmail, noEmailMessage);
                 }
             }else {
-                showNoEmailMessage(actionNeededEmail);
+                showNoEmailMessage(actionNeededEmail, noEmailMessage);
             }
 
         });
     }
 
     // Show the text field. Hide the "no email" message.
-    function showActionNeededEmail(actionNeededEmail){
-        let noEmailMessage = document.getElementById("no-email-message");
+    function showActionNeededEmail(actionNeededEmail, noEmailMessage){
```

Comment:
Nitpick: I think it's more readable if you got rid of these 2 simple functions and just called the show/hide